index.js

```diff
@@ -202,7 +202,16 @@ async function run(context, plugins) {
     logger.warn(`Skip ${nextRelease.gitTag} tag creation in dry-run mode`);
   } else {
     // Create the tag before calling the publish plugins as some require the tag to exists
-    await tag(nextRelease.gitTag, nextRelease.gitHead, { cwd, env });
+    await tag(
+      nextRelease.gitTag,
+      nextRelease.gitHead,
+      {
+        tagAnnotate: options.tagAnnotate,
+        tagSign: options.tagSign,
+        tagMessage: template(options.tagMessage)({ nextRelease }),
```

Comment:
I would make the `tagMessage` optional, and default to the `nextRelease.gitTag`.

Example use case: I want my release tags to be signed, but I don't care about the annotation message.